src/Tizen.NUI/src/public/XamlBinding/ResourceDictionary.cs

```diff
@@ -77,19 +79,25 @@ public Type MergedWith
         }
 
         /// <summary>
-        /// Gets or sets the URI of the merged resource dictionary.
+        /// Gets or sets the resource dictionary.
         /// </summary>
-        /// This will be public opened in tizen_5.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        /// Internal using, will never open.
         [EditorBrowsable(EditorBrowsableState.Never)]
         [TypeConverter(typeof(RDSourceTypeConverter))]
-        public Uri Source
+        public ResourceDictionary Source
         {
-            get { return source; }
+            get
+            {
+                return this;
+            }
             set
             {
-                if (source == value)
-                    return;
-                throw new InvalidOperationException("Source can only be set from XAML."); //through the RDSourceTypeConverter
+                OnValuesChanged(value.ToArray());
+
+                foreach (var pair in value)
```

Comment:
could you resolve the warning below?

**src/Tizen.NUI/src/public/XamlBinding/ResourceDictionary.cs#L97
In externally visible method 'void ResourceDictionary.Source.set', validate parameter 'value' is non-null before ...**